cmd/uplink/cmd/put.go

```diff
@@ -38,10 +38,10 @@ func putMain(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	// if uploading
-	if u0.Host == "" {
-		return fmt.Errorf("No bucket specified. Please use format sj://bucket/")
+	clp, err := fpath.New("-")
```

Comment:
Some better name here like `dst`, `output` or similar.

Reply:
Changed it everywhere in all commands.